setup.py

```diff
@@ -23,7 +23,7 @@
 
 
 setup(
-    name="teachers_digital_platform",
```

Comment:
It's above this reference (lines 19-22) but I don't think we need the `testing_extras` invocation if we're not testing python.

Reply:
Also the extras reference at line 58.

Reply:
I think we're going to keep stubs for testing Python so that any Python added to the cartoon can be tested.